src/components/Modals/OnboardingReview.vue

```diff
@@ -0,0 +1,514 @@
+          </button>
+        </span>
+      </div>
+      <div class="onboarding-inputs onboarding-inputs">
+        <div class="onboarding-subHeader2-fillRow">
+          <span class="onboarding-subHeader2-review"> Cornell Swimming Test</span>
+        </div>
+        <div class="onboarding-selectWrapper">
+          <div class="onboarding-selectWrapper-review">
+            <label class="onboarding-label">
+              <img class="checkmark" src="@/assets/images/checkmark-onboarding.svg" />
+              {{ this.user.tookSwim === 'yes' ? 'Yes' : 'No' }}
+            </label>
+          </div>
+        </div>
+        <div class="onboarding-subHeader2-fillRow">
+          <span class="onboarding-subHeader2-review"> Test Credits</span>
+        </div>
+        <div class="onboarding-selectWrapper">
+          <div class="onboarding-selectWrapper-reviewExam">
+            <div class="alignLeft">
+              <label class="onboarding-label">AP Credits</label>
+              <div v-for="(options, index) in displayOptions.exam" :key="'AP' + index">
+                <label
+                  v-if="typeof options.type != undefined && options.type.placeholder == 'AP'"
+                  class="onboarding-label--review"
+                  >{{ options.subject.placeholder }}</label
+                >
+              </div>
+              <label class="onboarding-label addSpaceTop">IB Credits</label>
+              <div v-for="(options, index) in displayOptions.exam" :key="'IB' + index">
+                <label
+                  v-if="typeof options.type != undefined && options.type.placeholder == 'IB'"
+                  class="onboarding-label--review"
+                  >{{ options.subject.placeholder }}</label
+                >
+              </div>
+            </div>
+            <div class="alignCenter">
+              <label class="onboarding-label">Score</label>
+              <div v-for="(options, index) in displayOptions.exam" :key="'APScore' + index">
+                <label
+                  v-if="typeof options.type != undefined && options.type.placeholder == 'AP'"
+                  class="onboarding-label--review"
+                  >{{ options.score.placeholder }}</label
+                >
+              </div>
+              <label class="onboarding-label addSpaceTop">Score</label>
+              <div v-for="(options, index) in displayOptions.exam" :key="'IBScore' + index">
+                <label
+                  v-if="typeof options.type != undefined && options.type.placeholder == 'IB'"
+                  class="onboarding-label--review"
+                  >{{ options.score.placeholder }}</label
+                >
+              </div>
+            </div>
+            <div class="alignCenter">
+              <label class="onboarding-label">Credit</label>
+              <div v-for="(options, index) in displayOptions.exam" :key="'APCredit' + index">
+                <!-- TODO replace credit with actual value -->
```

Comment:
What does this TODO mean?

Reply:
I think it was that the credit was computed based on a default value in the JSON without looking at the AP score, but if we remove the credit section like we discussed it shouldn't matter.